typescript/src/schema/comments.ts

```diff
@@ -121,13 +123,15 @@ export class CommentsMap {
                 ) {
                   paramType = field.typeName!;
                 }
+                const label = field.label ? field.label : undefined;
```

Comment:
Why not just `const label = field.label`? If it's undefined, it will be undefined.